packages/ui/src/components/AppLayout/index.tsx

```diff
@@ -183,6 +183,10 @@ const AppLayout: FC<PropsWithChildren<AppLayoutProps>> = ({
         ) : null;
     }, [splitPanelProps]);
 
+    useEffect(() => {
+        setNotifications(props.notifications);
+    }, [props.notifications]);
```

Comment:
Would it be simpler to get rid of the notifications from state render it directly through props rather than have the `useEffect` here? :)

Reply:
`notifications` is a context object to support bottom up change however there might be user cases the notifications are coming from top down.

Reply:
Makes sense! Thanks :)